src/resolvers/Query/getAllAgendaCategories.ts

```diff
@@ -0,0 +1,23 @@
+import type {
+  QueryResolvers,
+  AgendaCategory,
+} from "../../types/generatedGraphQLTypes";
+import { AgendaCategoryModel } from "../../models";
+import { errors, requestContext } from "../../libraries";
+import { AGENDA_CATEGORY_NOT_FOUND_ERROR } from "../../constants";
+/* eslint-disable */
+/**
+ * Resolver function for the GraphQL query 'agendaCategories'.
+ *
+ * This resolver fetches all agenda categories from the database.
+ *
+ * @returns {Promise<Array<AgendaCategory>>} A promise that resolves to an array of agenda categories.
+ * @throws {InternalServerError} Throws an error for potential issues during agenda categories fetching.
+ */
+/* eslint-enable */
+export const agendaCategories: QueryResolvers["agendaCategories"] =
```

Comment:
What is the point of this query?

Reply:
I made this so that a user could see all the categories of agendas an organization has, so he can create agendaItems on that basis. It would help a person creating a new agendaItem to check if his agendaItem does match any existing category.

Reply:
This is but returning the same across all the organizations and not a specific one. The same should be done via a field level resolver on the Organization model.

Reply:
Yes you are right, i am sorry. Now I see it does not actually have a point. I will delete this and add one for organization.